### ISSUE TITLE
Fix Request: IA11 AvoidParallelStreamWithCommonPool false negative on stream().parallel()

### ISSUE THREAD

**Issue by hwestra** [jborgers/PMD-jPinpoint-rules#243]
*IA11 AvoidParallelStreamWithCommonPool: parallelStream which uses the ForkJoinPool::commonPool is used*

There are 2 ways to create a parallel stream. Only one currently triggers rule IA11.

```
public class SomeClass {
    public void someMethod() {
    	var list = List.of(1,2);
        // triggers, as expected
        list.parallelStream().forEach(System.out::println);
        // no trigger
        list.stream().parallel().forEach(System.out::println);
    }
}
```

Please make `stream().parallel()` trigger as well.
